eclipse-bluechi/workspace/workloads/freyja.kube

```diff
@@ -16,5 +16,8 @@
 Yaml=freyja.yml
 
 # Commented to disable the service to automatically start
-# [Install]
-# WantedBy=default.target
+[Install]
+WantedBy=default.target
+
+[Unit]
+Wants=ibeji.service
```

Comment:
If more than one service can be listed, I would list ibeji and chariott

Reply:
well, Ibeji needs chariott, so you get that free by depending on ibeji